unittests/tools/test_veracode_sca_parser.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+
+from ..dojo_test_case import DojoTestCase
+from dojo.tools.veracode_sca.parser import VeracodeScaParser
+from dojo.models import Test
+
+from dateutil.tz import UTC
+
+
+class TestVeracodeScaScannerParser(DojoTestCase):
+
+    def test_parse_csv(self):
+        testfile = open("unittests/scans/veracode_sca/veracode_sca.csv")
+        parser = VeracodeScaParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(3, len(findings))
+
+        finding = findings[0]
+        self.assertEqual("Medium", finding.severity)
+        self.assertTrue(finding.active)
+        self.assertFalse(finding.is_mitigated)
+        self.assertEqual("AWS Java SDK for Amazon S3", finding.component_name)
+        self.assertEqual("1.11.951", finding.component_version)
+        self.assertEqual(1, len(finding.unsaved_vulnerability_ids))
+        self.assertEqual("CVE-2022-31159", finding.unsaved_vulnerability_ids[0])
+        self.assertEqual(6.4, finding.cvssv3_score)
+        self.assertEqual("36376", finding.vuln_id_from_tool)
+        self.assertEqual(datetime.datetime(2022, 7, 7, 9, 15, 0), finding.date)
+
+        finding = findings[1]
+        self.assertEqual("Medium", finding.severity)
+        self.assertTrue(finding.active)
+        self.assertFalse(finding.is_mitigated)
+        self.assertEqual("spring-cloud-function-context", finding.component_name)
+        self.assertEqual("3.2.5", finding.component_version)
+        self.assertEqual(1, len(finding.unsaved_vulnerability_ids))
+        self.assertEqual("CVE-2022-22979", finding.unsaved_vulnerability_ids[0])
+        self.assertEqual(5, finding.cvssv3_score)
+        self.assertEqual("36006", finding.vuln_id_from_tool)
+        self.assertEqual(datetime.datetime(2022, 6, 14, 11, 34, 0), finding.date)
+
+        finding = findings[2]
+        self.assertEqual("High", finding.severity)
+        self.assertFalse(finding.active)
+        self.assertTrue(finding.is_mitigated)
+        self.assertEqual("Apache Commons Configuration", finding.component_name)
+        self.assertEqual("2.1.1", finding.component_version)
+        self.assertEqual(1, len(finding.unsaved_vulnerability_ids))
+        self.assertEqual("CVE-2022-33980", finding.unsaved_vulnerability_ids[0])
+        self.assertEqual(7.5, finding.cvssv3_score)
+        self.assertEqual("36282", finding.vuln_id_from_tool)
+        self.assertEqual(datetime.datetime(2022, 7, 2, 23, 19, 0), finding.date)
+
+    def test_parse_json(self):
+        testfile = open("unittests/scans/veracode_sca/veracode_sca.json")
+        parser = VeracodeScaParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(1, len(findings))
+
+        finding = findings[0]
+        self.assertEqual("High", finding.severity)
+        self.assertTrue(finding.active)
+        self.assertFalse(finding.is_mitigated)
+        self.assertEqual("Apache Calcite Avatica", finding.component_name)
```

Comment:
```suggestion
        self.assertEqual("avatica-core", finding.component_name)
```
in the data it seems we have the data like this `maven:org.apache.calcite.avatica:avatica-core:1.11.0:` , in dojo we have only component name and version not the group. so it's better to set this data that will be aligned with others parsers.